charts/victoria-metrics-distributed/values.yaml

```diff
@@ -4,91 +4,131 @@ nameOverride: "vm-distributed"
-        topologyKey: kubernetes.io/hostname
-        whenUnsatisfiable: ScheduleAnyway
-    vmauthIngest:
-      # -- Create vmauth as a local write endpoint
+write:
+  global:
+    vmauth:
+      # -- Create a vmauth as the global write entrypoint
       enabled: true
       # -- Override the name of the vmauth object
       name: ""
-      # -- VMAuth spec. More options can be found [here](https://docs.victoriametrics.com/operator/api/#vmauthspec)
+      # -- Spec for VMAuth CRD, see [here](https://docs.victoriametrics.com/operator/api#vmauthspec)
       spec:
-        extraArgs:
-          # vmauth discovers backend IPs via periodic DNS queries,
-          # and performs load balancing between vminsert instances
-          # check https://docs.victoriametrics.com/vmauth/#discovering-backend-ips
-          discoverBackendIPs: "true"
-    vmauthQueryPerZone:
-      # -- Create vmauth as a local read endpoint
+        port: "8427"
+
+read:
+  global:
+    vmauth:
+      # -- Create vmauth as the global read entrypoint
       enabled: true
       # -- Override the name of the vmauth object
       name: ""
-      # -- VMAuth spec. More options can be found [here](https://docs.victoriametrics.com/operator/api/#vmauthspec)
+      # -- Spec for VMAuth CRD, see [here](https://docs.victoriametrics.com/operator/api#vmauthspec)
       spec:
-        extraArgs:
-          discoverBackendIPs: "true"
-    vmauthCrossAZQuery:
-      # -- Create a vmauth with all the zone with `allowQuery: true` as query backends
-      enabled: true
-      # -- Override the name of the vmauth object
-      name: ""
-      # -- VMAuth spec. More options can be found [here](https://docs.victoriametrics.com/operator/api/#vmauthspec)
-      spec: {}
-    # -- VMAgent here only meant to proxy write requests to each az,
+        port: "8427"
+
+# Config per availability zone components, including vmagent, vmcluster, vmauth etc
+availabilityZones:
+  -
+    # -- Availability zone name
+    name: zone-eu-1
+    common:
+      # -- Common for [VMAgent](https://docs.victoriametrics.com/operator/api/#vmagentspec),
+      # [VMAuth](https://docs.victoriametrics.com/operator/api/#vmauthspec),
+      # [VMCluster](https://docs.victoriametrics.com/operator/api/#vmclusterspec) spec params, like nodeSelector,
+      # affinity, topologySpreadConstraint, etc
+      spec:
+        nodeSelector:
+          topology.kubernetes.io/zone: zone-eu-1
+        affinity: {}
+          # preferredDuringSchedulingIgnoredDuringExecution:
+          # - weight: 1
+          #   preference:
+          #     matchExpressions:
+          #     - key: topology.kubernetes.io/zone
+          #       operator: In
+          #       values:
+          #       - us-central1-b
+        topologySpreadConstraints:
+          - maxSkew: 1
+            topologyKey: kubernetes.io/hostname
+            whenUnsatisfiable: ScheduleAnyway
+    write:
+      # -- Allow data ingestion to this zone
+      allow: true
```

Comment:
How about unifying them to `enable`?

Reply:
`enable` is for resource creation and `allow` is for traffic routing